website/src/components/testimonialSlider/interface.ts

```diff
@@ -0,0 +1,10 @@
+export interface Testimonials {
```

Comment:
No need to add the `Testimonials` interface. we can use `Testimonial[]` for our usecase

Reply:
its not working when the data passed as an component prop

Reply:
we can have an interface inside the component (`TestimonialSliderProps`) or in a separate file. both are fine. 😄